core/src/main/java/com/linecorp/armeria/common/auth/AuthUtil.java

```diff
@@ -14,16 +14,16 @@
  * under the License.
  */
 
-package com.linecorp.armeria.internal.common.util;
+package com.linecorp.armeria.common.auth;
 
 import javax.annotation.Nullable;
 
-public final class AuthUtil {
+final class AuthUtil {
```

Comment:
Could you explain why this class became a part of public API

Reply:
It's package-private so not a part of the public APIs. 😄
If you think it's better to put `secureEquals` method to the `BasicToken`, then I will do.

Reply:
Doh! Sorry about missing that out. :sweat: